### PR TITLE
Release - 2022-02-23

### PR DESCRIPTION
**Pull request by FSM1**
Release of latest to PROD

### CODE REVIEW

**Review by Tbaut — APPROVED** [review]
Tests are passing, nothing can go wrong :D

**Review by asnaith — APPROVED** [review]
There's lots of changes here 😅 but I've reviewed the commit history and performed some additional smoke testing on stage for things I was unsure about. 

With our whitelist, test automation suite, and manual testing at the PR stage, we are well covered from a test/risk perspective. 

Lots of nice improvements and fixes are included in here 😃

**Review by tanmoyAtb — APPROVED** [review]
This is happenin !!

File: packages/files-ui/src/locales/de/messages.po [hunk 1]
```diff
@@ -586,6 +748,9 @@ msgstr "Datei melden"
 msgid "Report a bug"
 msgstr "Fehler melden"
 
+msgid "Request access!"
+msgstr ""
+
```
Comment (Tbaut) [inline packages/files-ui/src/locales/de/messages.po:753]: I'm not sure what's wrong with the way we want to merge, but that seems to be the annoyance...

Comment (FSM1) [reply packages/files-ui/src/locales/de/messages.po:753]: ahhh dammit... no idea why there is a merge conflict... 🤔

Comment (Tbaut) [reply packages/files-ui/src/locales/de/messages.po:753]: I'll make a PR for this specifically.. let's see

Comment (Tbaut) [reply packages/files-ui/src/locales/de/messages.po:753]: ok that was a failed attempt, I wonder what would happen if we nuke this.
Will it make conflicts later too..
```suggestion
```

**Pull request merged**